spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -122,7 +115,6 @@ def assert_common
       assert_routers
       assert_specific_routers
       assert_specific_volumes
```

Comment:
did you want to remove assert_specific_volumes also, just like assert_specific_volume_snapshots?